src/relayclient/RelayClient.ts

```diff
@@ -164,22 +197,10 @@ export class RelayClient {
     return false
   }
 
-  async init (): Promise<this> {
-    await this.initMutex.runExclusive(async () => {
-      if (this.initialized) {
-        return this
-      }
-      this.emit(new GsnInitEvent())
-      await this.contractInteractor.init()
-      this.initialized = true
-    })
-    return this
-  }
-
   async relayTransaction (gsnTransactionDetails: GsnTransactionDetails): Promise<RelayingResult> {
     if (!this.initialized) {
       if (!this.initMutex.isLocked()) {
```

Comment:
Note that `initMutex` is not being locked any more, so this `isLocked ` check is not useful anymore

Reply:
yes, completelyremoved it.